{"stack_id": 366208, "component": "Main", "importance": "P5 normal", "reported_time": "2011-12-09 10:28", "modified_time": "2013-12-27 08:31", "stack_arr": [{"exception": "java.lang.AssertionError", "calls": [{"package": "org.eclipse.emf.facet.widgets.nattable.tests.swtbot", "class": "NatTableUITests", "method": "testAddQueryColumn", "filename": "NatTableUITests", "line": "218"}, {"package": "pde.test.utils", "class": "PDETestListener", "method": "testFailed", "filename": "PDETestListener", "line": "124"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient$7", "method": "run", "filename": "RemoteTestRunnerClient", "line": "551"}, {"package": "org.eclipse.core.runtime", "class": "SafeRunner", "method": "run", "filename": "SafeRunner", "line": "42"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient", "method": "notifyTestFailed", "filename": "RemoteTestRunnerClient", "line": "549"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient", "method": "access$17", "filename": "RemoteTestRunnerClient", "line": "544"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient$TraceProcessingState", "method": "readMessage", "filename": "RemoteTestRunnerClient", "line": "171"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient", "method": "receiveMessage", "filename": "RemoteTestRunnerClient", "line": "374"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient", "method": "access$27", "filename": "RemoteTestRunnerClient", "line": "373"}, {"package": "org.eclipse.jdt.internal.junit.model", "class": "RemoteTestRunnerClient$ServerConnection", "method": "run", "filename": "RemoteTestRunnerClient", "line": "271"}]}], "duplicated_stack_id": []}